feat(pkg-r): website improvements (#162)
posit-dev/querychat: pkg-py/docs/build.qmd

This is equivalent to the user asking the LLM to "reset" or "show all data".

## Multiple tables

Currently, you have two options for exploring multiple tables in QueryChat:

1. Join the tables into a single table before passing to QueryChat
2. Use multiple QueryChat instances in the same app

The first option makes it possible to chat with multiple tables inside a single chat interface, whereas the second option requires a separate chat interface for each table.

::: {.callout-note}
### Multiple filtered tables

> We do intend on supporting multiple filtered tables in a future release -- if you're interested in this feature, please up vote [the relevant issue](https://github.com/posit-dev/querychat/issues/6)

Reviewer: ```suggestion
We plan to support multiple filtered tables in a future release -- if you're interested in this feature, please up vote [the relevant issue](https://github.com/posit-dev/querychat/issues/6)
```

:::

Here's an example of the second approach, using two separate `QueryChat` instances to explore both the `titanic` and `penguins` datasets within the same app:

<details>

> <summary> <code>app.py </code> </summary>

Reviewer: Two quick things because we're using Quarto here:

1. We should put these little apps in their own source files and include them using `{{< include app-two-chats.py >}}`. That makes it much easier to run and test the apps outside of the Quarto document.
2. Instead of using `<details><summary>` elements, we can lean on Quarto's code folding. It's functionally similar but visually I think it will look better. If not, we could use callouts. The bare details elements don't look so great.
